subsys/nrf_security/src/drivers/cracen/Kconfig

```diff
@@ -5,7 +5,7 @@
 #
 
 config CRACEN_HW_PRESENT
-	def_bool SOC_SERIES_NRF54LX
+	def_bool (SOC_SERIES_NRF54LX || SOC_SERIES_NRF92X)
```

Comment:
This addition should not be needed I think - it should already be selected by the board